onfido-java/src/main/java/com/onfido/Onfido.java

```diff
@@ -0,0 +1,44 @@
+package com.onfido;
+
+import com.onfido.api.Resources;
+import com.onfido.managers.ApplicantManager;
+
+public final class Onfido {
+
+  private static final String DEFAULT_API_URL = "https://api.onfido.com/v3/";
+  private static final String US_API_URL = "https://api.us.onfido.com/v3/";
+
+  public final Config config;
+
+  public final ApplicantManager applicant;
+
+  private Onfido(Builder builder) {
+    config = new Config(builder);
+    applicant = new ApplicantManager(this);
```

Comment:
Rather than passing down `this` could you just pass down the `config`? I think that should be all the `Resources` class needs rather than the whole `onfido` object

Reply:
Done